packages/commons/src/auth/jwt.ts

```
@@ -41,60 +41,75 @@ export const verifyJwtToken = async (
   config: JWTConfig,
   logger: Logger
 ): Promise<{ decoded: JwtPayload | string }> => {
-  try {
```

Since all the logic is inside `jwt.verify` we don't need this `try/catch`

---

> if `buildJwksClients` fails we lose the log

You're right, I re-added the try/catch but with rejecting `tokenVerificationFailed` instead of the error (Which is still logged) in order to return more relevant informations to the caller, let me know if you agree